pkgs/os-specific/linux/nvidia-x11/default.nix

```diff
@@ -22,10 +22,10 @@ rec {
   # Policy: use the highest stable version as the default (on our master).
   stable = if stdenv.hostPlatform.system == "x86_64-linux"
     then generic {
-      version = "455.45.01";
-      sha256_64bit = "6tyMfggvZVQPp/aiSdgwn7VG/mIGb0lUcnAdyMEDoVM=";
-      settingsSha256 = "70ABqnO/heCp/5IztpU0Lo7eZd4n4wUoTlp1xIQ3aCc=";
-      persistencedSha256 = "36sM+djZvm77Gle7dcZ5tppgzQkD4IA0FJgCGsdZRI8=";
+      version = "460.27.04";
```

Comment:
isn't 460.27.04 a beta release?

Reply:
You are right, sorry for the oversight. I opened  #107105 to revert this.

Reply:
Sorry for that. It does not matter to me, I prefer the latest drivers, but I think it might lead to unexpected behavior with cuda.
 A few lines down you can replace
```
beta = stable;
```
with the beta build
```
beta = generic {
      version = "460.27.04";
      ....
}
```
then setting `services.xserver.videoDrivers = [ "nvidiaBeta" ];`

Also note  https://github.com/NixOS/nixpkgs/pull/99323 which would allow us to select driver package versions directly.